systems/plants/@RigidBodyManipulator/addRobotFromSDF.m

```diff
@@ -312,6 +328,9 @@
         if (options.visual || options.visual_geometry)
          geometry = RigidBodyGeometry.parseSDFNode(geomnode,xyz,rpy,model,body.robotnum,options);
          if ~isempty(geometry)
+           if isa(geometry,'RigidBodyMesh')
+             geometry.scale=.0254; % gazebo dae meshes appear to be in inches
+           end
```

Comment:
i suppose I will bring this in as is, and update it after we start loading more meshes.  the scaling is either correct or so ridiculously off that there is little chance of it hiding something.